src/wallet/node_rpc_proxy.cpp

```diff
@@ -36,18 +36,16 @@ using namespace epee;
 namespace tools
 {
 
-void NodeRPCProxy::init(const std::string &daemon_address)
-{
-  m_daemon_address = daemon_address;
-
-  m_height = 0;
-  m_height_time = 0;
-  for (auto &slot: m_earliest_height)
-    slot = 0;
-  m_dynamic_per_kb_fee_estimate = 0;
-  m_dynamic_per_kb_fee_estimate_cached_height = 0;
-  m_dynamic_per_kb_fee_estimate_grace_blocks = 0;
-}
+NodeRPCProxy::NodeRPCProxy(epee::net_utils::http::http_simple_client &http_client, boost::mutex &mutex)
+  : m_http_client(http_client)
+  , m_daemon_rpc_mutex(mutex)
+  , m_height(0)
+  , m_height_time(0)
+  , m_earliest_height()
```

Comment:
Yes. `m_earliest_height` is value intialized here. Value initialized arrays value initialize each element of the array. A value initialized integer is set to 0.